docs/widgets/progress_bar.md

```diff
@@ -106,27 +106,22 @@ Refer to the [section below](#styling-the-progress-bar) for more information.
 
 ## Reactive Attributes
 
-| Name | Type | Default | Description |
-|------|------|---------|-------------|
-| `percentage` | `float | None` | `None` | The read-only percentage of progress that has been made. This is `None` if the `total` hasn't been set. |
-| `progress` | `float` | `0` | The number of steps of progress already made. |
-| `total` | `float | None` | `None` | The total number of steps that we are keeping track of. |
-
-## Messages
-
-- [ProgressBar.Completed][textual.widgets.ProgressBar.Completed]
-- [ProgressBar.Started][textual.widgets.ProgressBar.Started]
+| Name         | Type    | Default | Description                                                                                             |
+| ------------ | ------- | ------- | ------------------------------------------------------------------------------------------------------- |
+| `percentage` | `float  | None`   | The read-only percentage of progress that has been made. This is `None` if the `total` hasn't been set. |
+| `progress`   | `float` | `0`     | The number of steps of progress already made.                                                           |
+| `total`      | `float  | None`   | The total number of steps that we are keeping track of.                                                 |
```

Comment:
Do you have a tool/plugin to format tables like this automatically?

Reply:
Yes, a vscode extension